assets/queries/k8s/saTokenSecretQuery/query.rego

```diff
@@ -0,0 +1,56 @@
+package Cx
+
+CxPolicy [result] {
+    document := input.document[i]
+    metadata := document.metadata
+    
+    validKind := ["Role","ClusterRole"]
+    ruleTaint := ["get","watch","list","*"]
+    resourcesTaint := ["secrets","pods"]
```

Comment:
Is necessary to prevent access to pods? From what I understand here we should only limit access to secrets